thisdata.gemspec

```diff
@@ -27,14 +27,15 @@ Gem::Specification.new do |spec|
   spec.executables   = []
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "httparty", "~> 0.13"
-  spec.add_runtime_dependency "json", "~> 2.3"
+  spec.required_ruby_version = '>= 2.4'
 
-  spec.add_development_dependency "bundler", "~> 1.10"
+  spec.add_runtime_dependency "httparty"
```

Comment:
Do we need to specify a version for these dependencies?

Reply:
I think not because in most updates of this gems we don't have critical changes and adding a specific version will just add extra work to update the library, but I can specify the latest version if you think it would be better

Reply:
Should probably peg to the minor version so that we don't have an issue with breaking changes on a major version update.